packages/lib/src/form-fields/slider.wc.svelte

```diff
@@ -0,0 +1,132 @@
+<svelte:options
+  customElement={{
+    tag: 'jp-slider',
+    shadow: 'none',
+    extend: (customElementConstructor) => {
+      return class extends customElementConstructor {
+        static formAssociated = true;
+
+        constructor() {
+          super();
+          this.attachedInternals = this.attachInternals();
+        }
+      };
+    }
+  }}
+/>
+
+<script lang="ts">
+  import { createEventDispatcher } from 'svelte';
+
+  export let disabled: boolean = false;
+  export let discrete: boolean = true; // true -> displays a numeric value
```

Comment:
"Discrete" is opposite of continuous. When `discrete` is `false`, `step` should be `'any'`.

For displaying numeric value, you should create another attribute called `showValue` or something similar.